src/components/TodoItem.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Form, Button, ListGroup } from 'react-bootstrap';
+
+const TodoItem = ({ todo }) => {
+  return (
+    <ListGroup.Item>
+      <Form className="d-flex align-items-center">
+        <Form.Check className="mr-2" type="checkbox" />
+        <Form.Label className="m-0">{todo}</Form.Label>
+        <Button variant="danger" type="submit" className="ml-auto">
+          Delete
+        </Button>
+      </Form>
+    </ListGroup.Item>
+  );
+};
+
+TodoItem.propTypes = {
+  todo: PropTypes.string,
+};
```

Comment:
用了 Typescript 的好處就是可以這樣寫，之後 vscode 還會提醒我們 code 是否有寫對

```suggestion
interface TodoItemProps {
  todo: string;
}

const TodoItem: React.FunctionComponent<TodoItemProps> = ({ todo }) => {
  return (
    <ListGroup.Item>
      <Form className="d-flex align-items-center">
        <Form.Check className="mr-2" type="checkbox" />
        <Form.Label className="m-0">{todo}</Form.Label>
        <Button variant="danger" type="submit" className="ml-auto">
          Delete
        </Button>
      </Form>
    </ListGroup.Item>
  );
};
```